gulpfile.js

```diff
@@ -632,6 +634,7 @@ function parseExtensionFlags() {
         log(minimalSetMessage);
         process.exit(1);
       }
+      argv.extensions = argv.extensions.replace(/\s/g, '');
```

Comment:
this is separated issue. added here since it's a one line change : )
to fix use case like `--extensions="amp-foo,  amp-bar"`

Reply:
Cool!